controllers/constant/odlm.go

```diff
@@ -855,6 +858,11 @@ spec:
             walStorage:
               size: 1Gi
         force: true
+        annotations:
+          k8s.enterprisedb.io/addons: ["velero"]
```

Comment:
This actually needs to be '["velero"]' otherwise it is invalid syntax

Reply:
This was already changed here https://github.com/IBM/ibm-common-service-operator/commit/c6afe100ce0dc5aa9a24c2a8ad5e2e01df3ca978